lib/model/connection.js

```diff
@@ -1,50 +1,44 @@
+var Lang = require('./../utilities/lang');
+var AttrValidators = require('./../utilities/attrValidators');
+var createClass = require('./../utilities/createClass');
+var GraphModelBase = require('./graphModelBase');
+
+var Port = require('./port');
+
 /**
- * connection.js
- * @author Salvador Abate
  * Definition for the Connection class. A Connection is the linkage between a
  * Node's Output and another (potentially different) node's Input.
  * 
  * Alternatively, in the situation where a node is contained within another node,
  * the containee can have a connection from one of its Outputs to one of the
  * container's Outputs. The same applies for Inputs.
- */
-
-/**
- * Creates a new connection.
+ * 
+ * @class Connection
  * @constructor
  *
- * @param id - The Connection's unique identifier.
- * @param from - The Output object that is the connection's source.
- * @param to - The Input object that is the connection's target.
- * @param metadata - A standard JavaScript object containing key-value metadata.
+ * @param config.id - The Connection's unique identifier.
```

Comment:
Sorry, I think I misunderstood the question.
It's not there because it's inherited from GraphBaseModel.